rskj-core/src/main/java/org/ethereum/rpc/Filter.java

```diff
@@ -32,13 +34,19 @@
  */
 
 public class Filter {
-    abstract static class FilterEvent {

```

Comment:
Just applied a Sonar suggestion to use interface instead of abstract, as it has more sense.